CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 - Pass OCR parameters properly in Qubes ([issue #455](https://github.com/freedomofpress/dangerzone/issues/455))
 - Fix dark mode support ([issue #550](https://github.com/freedomofpress/dangerzone/issues/550),
   thanks to [@garrettr](https://github.com/garrettr))
-- Fix issue where on Linux the update check (which exists so unsupported distros can still check for updates) once toggled would become unchecked upon restarting Dangerzone ([issue #513](https://github.com/freedomofpress/dangerzone/issues/513))
+- MacOS/Windows: when accepting checking for updates, the "check for updates" checkbox in the hamburger menu (top-right) would be unchecked, even though it is in fact going to check for updates ([issue #513](https://github.com/freedomofpress/dangerzone/issues/513))
```

Comment:
Hm, the wording is a bit complex, I'm afraid. What about?

```suggestion
- MacOS/Windows: Sync "Check for updates" checkbox with the user's choice ([issue #513](https://github.com/freedomofpress/dangerzone/issues/513))
```

Reply:
done in a89c496.